container/layer.bzl

```diff
@@ -103,6 +103,13 @@ def build_layer(
     args.add(directory, format = "--directory=%s")
     args.add(ctx.attr.mode, format = "--mode=%s")
 
+    if ctx.attr.mtime != -1:  # Note: Must match default in rule def.
```

Comment:
Could use a local private variable at the top of this file to do this automatically. So at the top of this file, consider declaring
`_DEFAULT_MTIME = -1`
and use it here and in the rule attribute definition.

Reply:
Great idea. Better than littering the code with duplicated magic numbers.

Reply:
Duplicated it upstream to rules_pkg too https://github.com/bazelbuild/rules_pkg/pull/125